Code/autopkglib/Unarchiver.py

```diff
@@ -28,6 +28,7 @@
     'zip': ['zip'],
     'tar_gzip': ['tar.gz', 'tgz'],
     'tar_bzip2': ['tar.bz2', 'tbz'],
+    'tar_xz': ['xz'],
```

Comment:
Shouldn't this be ['tar.xz', xz'] ? The original reports from thenikola referred to "tar.xz" and "tar.gz" extensions so it's not entirely clear.